snippets/perl/post_nb_cleanup_asset.pl

```diff
@@ -0,0 +1,75 @@
+#Load module netbackup.pm from current directory
```

Comment:
Should the name be post_nb_assets_cleanup?

Reply:
I will rename the file to post_nb_asset_cleanup.pl